packages/components/autocomplete/src/autocomplete.ts

```diff
@@ -28,63 +27,89 @@ export type AutocompleteFetchSuggestions =
     ) => Awaitable<AutocompleteData> | void)
   | AutocompleteData
 
+const { appendTo, placement, teleported } = useTooltipContentProps
+
 export const autocompleteProps = buildProps({
+  /**
+   * @description key name of the input suggestion object for display
+   * */
```

Comment:
Maybe we should use the same
```js
 /**
   * @description key name of the input suggestion object for display
   */
```

Reply:
I think it's good to be clear for consistency.